primitives/core/src/assertion.rs

```diff
@@ -53,7 +69,7 @@ pub enum Assertion {
 	A11(ParameterString),                                   // (minimum_amount)
 	A12(ParameterString),									// (target_twitter_screen_name)
 	A13(u32),                                               // (Karma_amount) - TODO: unsupported
-
+	A14(ParameterString, NativeToken, ParameterYears)	    // (min_balance, native_token, [years -> eg [2021, 2022]])
```

Comment:
I'm actually inclined to make it simple, because they are specifically made for the decoded event. Maybe not (yet) a good time to generalise it.

For the decoded event we know the starting date is 2023-01-01 (from feature request), so we can save one param.

The "native_token" can be parameterised, but then you need to include the token symbol in the final VC. Alternatively it can be omitted, and we simply have two VCs representing LIT and DOT respectively. 

For the building part we could simply copy a4 and a7, with the starting date fixated at 2023-01-01 and that's it :P
What do you think?

Reply:
A even more lazy way is not to have new assertions at all - we simply use the old a4 and a7. The VC verifier will decide if it meets the requirement 😂

Reply:
If you feel that current a4 and a7 are sufficient then let's use them.
The situation I want to avoid is adding and then removing assertions, or keeping assertions that are not going to be used anymore.

Reply:
Let me double-check it with Kayla
I'm personally fine with adding and removing assertions, as long as they are trivial and don't cost too much effort

Reply:
I have talked with @kaylawangnow, let's reuse A4/A7 for simplicity.

It would be however great if we can still test it - like find an account that started to hold before 2023-01-01 and check the generated VC.

Reply:
I think the token could be included this way:

```
"assertions": [
      {
        "and": [
          {
            "src": "$minimum_amount",
            "op": "==",
            "dst": "1"
          },
          {
            "src": "$token",
            "op": "==",
            "dst": "LIT"
          },
          {
            "src": "$from_date",
            "op": "<",
            "dst": "2022-01-01"
          },
          {
            "src": "$to_date",
            "op": ">=",
            "dst": "2022-12-31"
          }
        ]
      }
    ],
   
```

Reply:
You mean the merge of the HOLDER VCs. Yes it should be possible, most likely we'll do it after the decoded event.

cc @BillyWooo @zhouhuitian

Reply:
I created the issue: https://github.com/litentry/litentry-parachain/issues/1813

Reply:
> You mean the merge of the HOLDER VCs. Yes it should be possible, most likely we'll do it after the decoded event.
> 
> cc @BillyWooo @zhouhuitian

Not really, I just posted possible solution to: _The "native_token" can be parameterised, but then you need to include the token symbol in the final VC_